src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1399,12 +1412,16 @@ export default class CanvasController {
         }
 
         // Text
-        const gap = 3;          // height of TOTAL vertical space between the rows (0 for touching)
+        const gap = 3; // height of TOTAL vertical space between the rows (0 for touching)
         const lineheight = 4.5; // height of text row (used for spacing basis)
         const row1text = callsign;
         const aircraftAltitude = round(aircraftModel.altitude * 0.01);
         const aircraftSpeed = round(aircraftModel.groundSpeed * 0.1);
-        const row2text = `${leftPad(aircraftAltitude, 3)} ${leftPad(aircraftSpeed, 2)}`;
+        let row2text = `${leftPad(aircraftAltitude, 3)} ${leftPad(aircraftSpeed, 2)}`;
+
+        if (this.shouldShowSecondaryDataBlock) {
+            row2text = `${aircraftModel.model.icao.toUpperCase()} ${aircraftModel.fms.currentWaypoint.name}`;
```

Comment:
The ticket says "the scratchpad and aircraft type". The implementation is "aircraft type - scratchpad". @erikquinn